lib/Normalize_extended_ast.ml

```diff
@@ -50,7 +50,8 @@ let normalize_parse_result ast_kind ast comments =
     comments
 
 let normalize_code conf (m : Ast_mapper.mapper) txt =
-  match Parse_with_comments.parse_toplevel conf ~source:txt with
+  let input_name = "<input>" in
```

Comment:
Normalize is not called on user's input but on the formatting step's output, which is also the output to the user. What about `<output>` ?

Reply:
done